lib/presentation/edit_workout/ParamContainer.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:dataclass/dataclass.dart';
+
+@dataClass
+class ParamContainer {
+  final int _newWeight;
+  final int _newRepCount;
+  final String _workoutId;
+  final String _workoutExerciseId;
+
+  ParamContainer(this._newWeight, this._newRepCount, this._workoutId,
+      this._workoutExerciseId);
+
+  int get newWeight => _newWeight;
```

Comment:
Can we remove the getters and make the fields public? Dataclass fields should be public :P